tests/tests/tier0/bluechi-agent-connect-via-controller-address/test_bluechi_agent_connect_via_controller_address.py

```diff
@@ -63,6 +63,9 @@ def test_agent_invalid_configuration(
     bluechi_node_default_config: BluechiAgentConfig,
     bluechi_ctrl_default_config: BluechiControllerConfig,
 ):
+    # Disable default local agent on controller node using UDS,
```

Comment:
Not related to this PR, but would be great to fix it here:
The name of the function should reflect the file name - so `test_agent_invalid_configuration` should be renamed to `test_bluechi_agent_connect_via_controller_address`.

Reply:
done, updated